org.jacoco.build/pom.xml

```diff
@@ -553,6 +558,31 @@
         </executions>
       </plugin>
 
+      <plugin>
+        <groupId>com.diffplug.spotless</groupId>
+        <artifactId>spotless-maven-plugin</artifactId>
+        <configuration>
+          <java>
+            <lineEndings>UNIX</lineEndings>
+            <includes>
+              <include>src/**/*.java</include>
+            </includes>
+            <eclipse>
+              <version>4.12.0</version>
+              <file>../org.jacoco.core/.settings/org.eclipse.jdt.core.prefs</file>
+            </eclipse>
+          </java>
+        </configuration>
+        <executions>
+          <execution>
+            <phase>prepare-package</phase>
```

Comment:
The license check we do in phase `verify`. Why do we use different phase here? Shouldn't be both in `verify` oder even `validate`?

Reply:
@marchof in fact maybe we should reconsider phase used for check of license header - my rationale for usage of `prepare-package`:

Localy I often use `mvn test` just to do a quick test of something, what not necessarily will be committed and might be removed right after. In this case failure of license check seems unnecessary and bothers sometimes.

Also localy I often use `mvn package` to not execute time consuming Maven integration tests. 

So decided to use `prepare-package` phase in order to not complicate first case by check of formatting and in order to not miss it in second case.

Reply:
@Godin Then we move the license check in a separate PR?

Reply:
@marchof yup